README.md

```diff
@@ -89,6 +89,7 @@ This is the container component that encapsulates the lanes and cards
 | cardDraggable       | boolean  | Set to false to disable card dragging. Default: true                                                                           |
 | collapsibleLanes    | boolean  | Make the lanes with cards collapsible. Default: false                                                                          |
 | editable            | boolean  | Makes the entire board editable. Allow cards to be added or deleted Default: false                                             |
+| canAddLanes         | boolean  | Allows new lanes to be added to the board.                          Default: false                                             |
```

Comment:
Can call out that this prop is relevant only when editable is true

Reply:
To be honest, I'm not a big fan of the `editable` prop, and would ideally like to phase it out, b/c it's pretty redundant: If `canAddLanes` is `true`, then that's meaning I want to be able to add lanes. Why do I need to set another prop (`editable`) to also be `true` for that to happen?

It also results in this situation: Since this feature was originally tied to just `editable`, I couldn't turn it off unless I made the entire board not editable.

In my opinion, it's far better to have each feature tied to their own prop, to allow for more fine-control over the boards features. 

I can understand this prop making sense in the early days, when the feature set was limited, but I think this library has outgrown the `editable` prop.